pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -30,6 +31,7 @@ import (
 
 const (
 	machineAnnotationKey = "machine.openshift.io/machine"
+	annotationReboot     = "metal3.io/BareMetalHostReboot"
```

Comment:
should we have a generic annotation key and not specific to metal3?

Reply:
@vikaschoudhary16 Does `machineremediation.openshift.io/reboot` suit well?